esp/scm/ws_logaccess.ecm

```diff
@@ -135,6 +160,7 @@ ESPRequest GetLogsRequest
     [min_ver("1.01")] ESPenum LogSelectColumnMode SelectColumnMode(1);
     ESParray<string> Columns;
     LogAccessLogFormat Format("JSON");
+    [min_ver("1.03")] ESParray<ESPstruct SortCondition> SortBy;
```

Comment:
@GordonSmith this allows you to specify multiple sortby conditions consisting of a columntype (bydate, byjobid, etc) and a direction.